benchmarks/main.swift

```diff
@@ -8,100 +8,100 @@ import PNG
 
 func structuredARGBPremultiplied(_ path:String) -> Int
 {
-    let t1:Int = clock()
+    let t1:Int = Int(clock())
```

Comment:
This is your code. I wasn’t really paying attention to what it does. All I did was let Xcode auto‐fix the errors and then checked whether it would build afterward.

Continuous integration now includes [building and running the tests on macOS](https://travis-ci.org/kelvin13/png/jobs/453715419#L174-L1221), so you can rearrange these to your heart’s content without fear of unknowingly breaking anything.

You do have write access to this branch, don’t you?

***

P.S. `Int(bitPattern: UInt)` ends up with a different value because their memory representation differs.

Reply:
I usually just use the reinterpret cast if I don’t expect the high bit to be set lol
I don’t have write access to the branch, it is your fork

Reply:
> I don’t have write access to the branch, it is your fork

That’s odd...

“Allow edits from maintainers.” *is* checked.

I can still do the adjusting myself if that’s what you would prefer, but I won’t have time for a day or too.

Reply:
Name ambiguity stuff was happening when tried to revisit `clock()`.

Why don’t you rearrange the OS‐compatibility the way you like it afterward in a separate pull request? Its current state is at least an improvement because it does compile successfully. (The `Int` → `Int` conversions on Linux should be erased by the optimizer anyway.)